submitted_models/ctu_cras_norlab_marmotte_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,1114 @@
+          <horizontal_fov>1.3</horizontal_fov>
+          <image>
+            <width>1920</width>
+            <height>1200</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.4</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>6</update_rate>
+        <pose frame="">0.22978 0.31824 0.08992 0 -0.26 1.88</pose>
+        <visualize>1</visualize>
+      </sensor>
+
+      <sensor name="camera_4" type="camera">
+        <camera name="camera_4">
+          <horizontal_fov>1.3</horizontal_fov>
+          <image>
+            <width>1920</width>
+            <height>1200</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.4</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>6</update_rate>
+        <pose frame="">0.38916 0.26483 0.08992 0 -0.26 0.58</pose>
+        <visualize>1</visualize>
+      </sensor>
+
+      <sensor name="camera_5" type="camera">
+        <camera name="camera_5">
+          <horizontal_fov>1.3</horizontal_fov>
+          <image>
+            <width>1920</width>
+            <height>1200</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.4</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>6</update_rate>
+        <pose frame="">0.16561 0.19955 0.20529 0 -1.57 0.0</pose>
+        <visualize>1</visualize>
+      </sensor>
+
+      <sensor name="camera_realsense" type="rgbd_camera">
+        <camera name="camera_realsense">
+          <horizontal_fov>1.5184</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <fx>337.22195</fx>
+              <fy>337.22195</fy>
+              <cx>320.5</cx>
+              <cy>240.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+          <image>
+            <width>640</width>
+            <height>480</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.4</near>
+            <far>300</far>
+          </clip>
+          <depth_camera>
+            <clip>
+              <near>0.4</near>
```

Comment:
The camera clipping was originally set to 0.07 which is the size of the camera lens mesh, and it worked correctly in the classical gazebo. In ignition, this value needs to be higher, because even with 0.1 or 0.2, the camera sometimes renders the robot mesh it is supposed to clip. We have observed that this depends on the orientation and velocity of the robot in the world frame, so it is probably some graphics bug.

We've observed the same behavior with MARV and our other robots. I just didn't yet have time to properly examine the issue and report it.